src/components/Quiz.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+import Answers from './Answers';
+import { publish } from './Event';
+import QuestionAnswer from './QuestionAnswer';
+import Score from './Score';
+import TimerContainer from './TimerContainer';
+
+function Quiz(props) {
+  const { quiz } = props;
+  const [questionNumber, setQuestionNumber] = useState(1);
+  const [questions, setQuestions] = useState([]);
+  const nextQuestion = (question) => {
+    if (questionNumber < (Number(quiz.noOfQuestionsVal) + 1)) {
+      setQuestions([...questions, question]);
+      publish(`ResetInterval${quiz.quizId}`, quiz.timerVal);
+      setQuestionNumber(questionNumber + 1);
+    }
+  };
+  const shouldShowQuiz = questionNumber < (Number(quiz.noOfQuestionsVal) + 1);
+  return (
+    <div>
+      {shouldShowQuiz && (
```

Comment:
The visibility fo timercontainer and questionanswer is dependent on shouldshowquiz. Can be moved into one bracket instead of separate for each.

Reply:
Done